README.md

```diff
@@ -106,18 +107,16 @@ vmlinuz-0.7.4
 vmlinuz-0.7.5
 ```
 
-
-With this layout, you'll now need a way to boot the kernel and initramfs via EFI. This can be done via a manual entry set via efibootmgr, or it can be done with rEFInd.
+With this layout, you'll now need a way to boot the kernel and initramfs via EFI. This can be done via a manual entry set via [efibootmgr](https://github.com/rhboot/efibootmgr), or it can be done with [rEFInd](http://www.rodsbooks.com/refind/).
 
 If you are using a pre-built kernel and initramfs downloaded from the releases page, you'll need to identify the following additional details:
 
-* Your system's hostid (`hostid`). It's important that this command is executed as root, to ensure that it returns the correct value.
+* Your system's hostid (`hostid`). It's important that this command be executed as root, to ensure that it returns the correct value.
```

Comment:
We no longer ship pre-built releases, so this language can probably be stripped out.